knotx-core/pom.xml

```diff
@@ -35,9 +35,10 @@
 
   <modules>
     <module>knotx-common</module>
-    <module>knotx-server</module>
     <module>knotx-repository</module>
-    <module>knotx-template-engine</module>
+    <module>knotx-server</module>
+    <module>knotx-service-adapter</module>
     <module>knotx-standalone</module>
+    <module>knotx-template-engine</module>
```

Comment:
The order of modules in pom is not important. We could sort them alphabetically.
However order of dependencies matters.